src/templater.rs

```diff
@@ -481,18 +508,23 @@ impl FormattedCommitOrChangeId<'_> {
 
     pub fn short(&self) -> String {
```

Comment:
nit: maybe rename this to `hex()`, and make the original `hex()` to private?

Reply:
Done.